swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, fabric="Unknown", condition=0, age=0):
```

Comment:
We usually want keep our parameters in the same order of parameters used by the parent class. Other, this is perfect! Great work!
```suggestion
    def __init__(self, id=None, condition=0, age=0, fabric="Unknown"):
```

**Note: This comment applies to all of your children classes**

Reply:
Didn't know about it, thanks!